server/index.js

```diff
@@ -70,6 +70,11 @@ const upload = multer({ dest: "./public/uploads/" });
 const PORT = 4000;
 const dbName = "ParkGames";
 
+// LOGGER.
+
+let logger = new Logger(process.env.LOG_LEVEL);
```

Comment:
We wanna call this before the imports above ^, if they call `getInstance()` then `new Logger` will throw an error